composer.json

```diff
@@ -14,9 +14,9 @@
   "authors": [],
   "require": {
     "psr/log": "^1.0.0 || ^3.0.0",
-    "php": ">=7.3",
+    "php": "^7.3 || ^8.0",
```

Comment:
What if someone needs php 7.4?

Reply:
It's marked with `^`. It's the same as `>=7.3 <= 9` I believe. You can validate with server checker

Reply:
Ok, my bad, this works with 7.4